components/Home/CustomerCard.js

```diff
@@ -0,0 +1,30 @@
+import { IoIosQuote } from "react-icons/io";
+
+export default function CustomerCard({ img, name, position, feedback }) {
+  return (
+    <div>
+      <div className="bg-white text-cyan_blue 	rounded-xl p-4 shadow-lg mx-8 lg:p-6 2xl:p-12">
+        <div className="flex items-center border-b-4 border-gray-50 mb-5 p-2">
+          <img
+            className="mr-4 2xl:mr-8 w-20 rounded-full"
+            src={img.src}
+            alt="Picture of the author"
+          />
```

Comment:
reduce the size of the image it's taking too much space and reduce the spacing between items so that an individual card can be bigger. Plus, there is some leading left padding before the first card, remove it. It has to be same alignment with above text and heading. The cards are not the same height. You can check the jobTypeCard how to make them same height, I think by just making them to flex solves the issue.